core/src/main/java/org/geysermc/geyser/translator/protocol/bedrock/entity/player/BedrockActionTranslator.java

```diff
@@ -201,35 +218,32 @@ public void translate(GeyserSession session, PlayerActionPacket packet) {
                 }
 
                 Vector3f vectorFloat = vector.toFloat();
-                LevelEventPacket continueBreakPacket = new LevelEventPacket();
-                continueBreakPacket.setType(LevelEvent.PARTICLE_CRACK_BLOCK);
-                continueBreakPacket.setData((session.getBlockMappings().getBedrockBlockId(breakingBlock)) | (packet.getFace() << 24));
-                continueBreakPacket.setPosition(vectorFloat);
-                session.sendUpstreamPacket(continueBreakPacket);
 
                 // Update the break time in the event that player conditions changed (jumping, effects applied)
                 LevelEventPacket updateBreak = new LevelEventPacket();
                 updateBreak.setType(LevelEvent.BLOCK_UPDATE_BREAK);
                 updateBreak.setPosition(vectorFloat);
-                double breakTime = BlockUtils.getSessionBreakTime(session, BlockState.of(breakingBlock).block()) * 20;
 
+                Direction direction = Direction.VALUES[packet.getFace()];
+                spawnBlockBreakParticles(session, direction, vector, BlockState.of(session.getGeyser().getWorldManager().getBlockAt(session, vector)));
```

Comment:
```suggestion
                spawnBlockBreakParticles(session, direction, vector, BlockState.of(breakingBlock));
```
Any reason this wouldn't work?
If it does, this could be a variable & also be used when calculating the block break time below

Reply:
Resolved in latest commit.